packages/bitcore-wallet-service/src/config.ts

```diff
@@ -476,12 +478,36 @@ const Config = (): any => {
   };
 
   // Override default values with bws.config.js' values, if present
+  const { BITCORE_CONFIG_PATH = '../../', BWS_CONFIG_PATH = '../../' } = process.env;
+  let bwsConfig;
+
   try {
-    // eslint-disable-next-line @typescript-eslint/no-require-imports
-    const bwsConfig = require('../../bws.config');
-    defaultConfig = _.merge(defaultConfig, bwsConfig);
-  } catch {
-    logger.info('bws.config.js not found, using default configuration values');
+    if (BITCORE_CONFIG_PATH) {
+      if (fs.existsSync(path.join(BITCORE_CONFIG_PATH, 'bitcore.config.json'))) {
+        bwsConfig = JSON.parse(fs.readFileSync(path.join(BITCORE_CONFIG_PATH, 'bitcore.config.json'), 'utf8'));
+      } else if (fs.existsSync(BITCORE_CONFIG_PATH) && BITCORE_CONFIG_PATH.endsWith('.json')) {
+        bwsConfig = JSON.parse(fs.readFileSync(BITCORE_CONFIG_PATH, 'utf8'));
+      }
+      bwsConfig = bwsConfig?.bitcoreWalletService;
+    }
+
+    if (!bwsConfig && BWS_CONFIG_PATH) {
+      if (fs.existsSync(path.join(BWS_CONFIG_PATH, 'bws.config.js'))) {
+        // eslint-disable-next-line @typescript-eslint/no-require-imports
+        bwsConfig = require(path.join(BWS_CONFIG_PATH, 'bws.config.js'));
```

Comment:
In testing your this, I realized it doesn't actually work for `bws.config.js` because of the discrepancy between how `require` and `fs.existsSync` handle the CWD. If I run bws from `bitcore/packages/bitcore-wallet-service`, then `fs.existsSync` will use a CWD of `bitcore/packages/bitcore-wallet-service` but the `require` uses a CWD of `bitcore/packages/bitcore-wallet-service/ts_build/src`. 

One way to fix this would be to use a "standard" starting point such as `__dirname` like so:
```javascript
const {
    BITCORE_CONFIG_PATH = path.join(__dirname, '../../../../'),
    BWS_CONFIG_PATH = path.join(__dirname, '../../')
  } = process.env;
```

Reply:
This is still an issue